web-common/tailwind.config.cjs

```diff
@@ -8,7 +10,22 @@ module.exports = {
   /** Once we have applied dark styling to all UI elements, remove this line */
   darkMode: "class",
   theme: {
-    extend: {},
+    colors: {
+      ...colors,
+      blue: {
+        50: "var(--color-primary-50)",
+        100: "var(--color-primary-100)",
+        200: "var(--color-primary-200)",
+        300: "var(--color-primary-300)",
+        400: "var(--color-primary-400)",
+        500: "var(--color-primary-500)",
+        600: "var(--color-primary-600)",
+        700: "var(--color-primary-700)",
+        800: "var(--color-primary-800)",
+        900: "var(--color-primary-900)",
+        950: "var(--color-primary-950)",
+      },
```

Comment:
The issue with this is we will have to have a toggle for any common components we create further down the line. Also blue-* will be changed to indigo-* for the filter pills. I feel the current approach is better until we are stable with colors.

Reply:
I'm not following – what do you mean "toggle for any common components"?

Even if we use `blue` in fewer places throughout the app, it's confusing for `blue` to implicitly mean `theme-primary-color`. The code (e.g. `bg-blue-500`) should say what it does, else it increases the cognitive burden on everyone reading it.

Reply:
I am not disagreeing we should not do it. I am saying once everything is stable we can fix it across the app. Right now virtual table uses blue which is used in dimension display. Updating that component to be aware of whether it is in dsahboard or not (just a color argument) feels like it will be some amount of throw away work if we decide to move away from dimension display. That is why we should hold off on that.